src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -287,7 +287,7 @@ public function buildView(FormView $view, FormInterface $form, array $options)
             $view->vars['placeholder'] = $options['placeholder'];
         }
 
-        if ($options['multiple'] && !$options['expanded']) {
+        if ($options['multiple'] && !$options['expanded'] && isset($view->vars['full_name'])) {
```

Comment:
I was unsure about adding this but it can easily be reverted. It seems to me it can't hurt to add this check and as PHPUnit tests seem to pass 🤷🏻 Anyway, I replicated the psalm setup from the Github action locally using Docker and psalm came up with this. Not sure whether it is something my baseline missed or because of changes I made.

Reply:
I think it's not needed. If I'm wrong, this should be patched in another PR anyway. So yes, please revert.